### PR TITLE
Doxygen code converted for STEER/AOD/

### PR DESCRIPTION
**Pull request by mikesas**

### CODE REVIEW

File: STEER/AOD/AliAODAD.cxx [hunk 1]
```diff
@@ -13,16 +13,12 @@
...
-//-------------------------------------------------------------------------
-//     Container class for AOD AD data
-//     Author: Michal Broz
-//     michal.broz@cern.ch
-//-------------------------------------------------------------------------
-
 #include "AliAODAD.h"
 #include "AliLog.h"
 
+/// \cond CLASSIMP
```
Comment (dberzano) [inline STEER/AOD/AliAODAD.cxx]: The `\cond` fence for `ClassImp` and `ClassDef` is no longer necessary, please remove :-)

**Comment by dberzano** [comment]
+1

**Comment by mikesas** [comment]
Done with the request of removing the fence around ClassImp and ClassDef, thanks again.

File: STEER/AOD/AliAODTrack.h [hunk 2]
```diff
@@ -322,7 +322,7 @@ class AliAODTrack : public AliVTrack {
   Double_t  GetTPCsignalTunedOnData() const { return fTPCsignalTuned;}
   void      SetTPCsignalTunedOnData(Double_t signal) {fTPCsignalTuned = signal;}
   UShort_t  GetTPCsignalN()      const { return fDetPid?fDetPid->GetTPCsignalN():0;    }
-  virtual Bool_t GetTPCdEdxInfo( AliTPCdEdxInfo &v) const {return fDetPid?fDetPid->GetTPCdEdxInfo(v):0;}
+  virtual AliTPCdEdxInfo* GetTPCdEdxInfo() const {return fDetPid?fDetPid->GetTPCdEdxInfo():0;}
```
Comment (dberzano) [inline STEER/AOD/AliAODTrack.h]: Hello @mikesas, after solving a build problem (triggered by a Clang warning, see alisw/AliRoot#583) I have noticed that... the problem was effectively triggered by this line.

Why have you changed _code_ in a PR mentioning changes only in Doxygen? Is this change needed? Could you please clarify? This change should be reverted, then go in a separate pull request in order to be discussed separately...

**Pull request merged**